nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractDatabaseFetchProcessor.java

```diff
@@ -161,7 +161,7 @@
                 .description("The type/flavor of database, used for generating database-specific code. In many cases the Generic type "
                         + "should suffice, but some databases (such as Oracle) require custom SQL clauses. ")
                 .allowableValues(dbAdapters.keySet())
-                .defaultValue(dbAdapters.values().stream().findFirst().get().getName())
+                .defaultValue("Generic")
```

Comment:
@mattyb149 In case your curious, I found that `Generic` may or may not be selected as the default after you start adding new DBAdapters. I ran into this exact issue when I added a new `Hive` DBAdapter, so I just hard coded it rather then having the default value possibly change.